private/bufpkg/bufstudioagent/plain_post_handler.go

```diff
@@ -222,6 +203,31 @@ func (i *plainPostHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func connectClientOptionsFromContentType(contentType string, requestMsgSize int) ([]connect.ClientOption, error) {
```

Comment:
No need for `requestMsgSize int` anymore.

Reply:
😬 🤦🏼‍♀️